templates/_footer.phtml

```diff
@@ -3,7 +3,7 @@
                 <div class="col-md-12">
                     <hr/>
                     <p>
-                        Copyright &copy; 2018 The Godot Engine community - MIT licensed
+                        Copyright &copy; <?php echo date("Y"); ?> The Godot Engine community - MIT licensed
```

Comment:
Can you change it to be `echo esc(date("Y"))` or `echo raw(date("Y"))`, so that grepping for `echo (?!esc|raw|url)` returns no results?

Reply:
Sure thing!

Maybe I'm just not as familiar with PHP, but what do `esc()` and `raw()` do? I can't seem to find any documentation around them?

Reply:
They are helper functions defined in `_header.phtml`. `esc()` escapes user input, while `raw()` outputs things as is and is really just for completions sake.

Reply:
Ahh, thanks for the clarification!